tests/integration_tests/multi_tables_ddl/data/test.sql

```diff
@@ -66,4 +108,16 @@ insert into t3 select * from t1_7;
 insert into t4 select * from t2_7;
 drop table t3, t4;
 
+/* replicate successful */
+rename table t9 to t99;
```

Comment:
Please add all possible cases to this test as we can since they are limited, like:
`rename t1 to t3`
`rename t44 to t4, t5 to t55`